sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiverClient.cs

```diff
@@ -974,7 +974,7 @@ internal Outcome GetDeferOutcome(IDictionary<string, object> propertiesToModify)
         /// If processing of the message requires longer than this duration, the lock needs to be renewed.
         /// For each renewal, it resets the time the message is locked by the LockDuration set on the Entity.
         /// </remarks>
-        public virtual async Task<DateTime> RenewLockAsync(
+        internal virtual async Task<DateTime> RenewLockAsync(
             string lockToken,
             CancellationToken cancellationToken = default)
         {
```

Comment:
Why are we just setting the `internal` and not removing this method?

Reply:
Removed